### DISCUSSION TITLE
Which ontologies have "adopted" terms from other ontologies?

### DISCUSSION THREAD

**Discussion by matentzn** [OBOFoundry/OBOFoundry.github.io#1923] [General]
We are interested to understand the extent to which ontologies have adopted _individual_ terms from other ontologies. For example, the relation ontology RO has adopted the `part of` and `occurs in` relationships from BFO: `BFO:0000050` and `BFO:0000066`.

**NOTE**: This is different from ontologies adopting an entire namespace: GO for example has GO_REL, and a few more namespace incorporated, but they have adopted the entire namespace, rather than just a set of individual terms. In other words, we still know exactly, based on the IRI of the term where it belongs - we cant say the same of `BFO:0000050`, which does _not_ belong in BFO (but RO).

Do you know ontologies that have done the same? Can you give some examples?

**Comment by cthoyt** [comment]
Lots of ontologies duplicate RO relations (some xref them back, but most don't). This is quite a systematic problem across OBO ontologies, and probably isn't the most interesting case of what you're looking for.

**Reply by matentzn** [reply]
These are bugs and need to be fixed, but they are not "adoptions" in the sense of this question - we care about the case where an _IRI_ "belongs" to some ontology seemingly but actually belongs somewhere else. In the case you mention, the relation from RO should be used, but due to historical reasons, they accidentally mint their own IRI.

**Comment by cmungall** [comment]
This makes complete sense

**Reply by ddooley** [reply]
This is about term resolution to the ontology of origin/maintenance?!  I'll just point out a utopian thought: An ultimate future solution is where term identifiers are minted in the form of generic ISBN_XYZ ids, and a purl system resolves each to its current maintainer (BFO, RO, etc curation teams). In this way term maintenance gets transferred from team to team as needed without changing the identifier.

**Comment by matentzn** [comment]
From @bpeters42 

> OBI has adopted several CHMO terms, as that ontology was no longer maintained. There have been ongoing complexities arising from this, and I am not so sure anymore that this is the right approach rather than just obsoleting a term with a 'replaced_by' mapping to the new term. It makes it complicated for example to know that the CHMO terms are part of the 'base' release of OBI.

**Comment by jamesaoverton** [comment]
(Sorry for the slow reply.)

There's a very small number of BFO terms that the OBO PURL system redirects to RO, for complex historical reasons: https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/config/obo.yml#L22

As Bjoern said, OBI "adopted" three CHMO terms: CHMO:0000087, CHMO:0000089, and CHMO:0000102. We could obsolete these IDs and mint new OBI IDs. More context: OBI didn't want to change the meaning of these terms, we just wanted to include them in OBI. CHMO imports from OBI, but I wanted to avoid a circular import from CHMO. OBI has circular imports with IAO that cause lots of problems. We got permission from the CHMO developers to "adopt" these three terms, rather than creating duplicate terms. At the time CHMO was not being maintained or was in hibernation, but I see that has changed. We did not change the PURL system for this. TLDR: This is not an example of best practise, but it is an example of a tough case we need to handle. We're open to changing how we handle these CHMO terms.

**Reply by matentzn** [reply]
Given that we are talking so far about 3 CHMO and maybe 5 BFO OPs, should we try to push for a rule:

Every IRI base (namespace) must belong to one and only one ontology. 

Explanation: it must be possible to determine the membership of an ontology entity to an ontology by looking at the URI prefix/namespace etc. 

Motivation: This will make it much easier for us to make enforceable rules around axiom injection, quality control checking across OBO (dashboard!), base extractions I personally volunteer to deal with the OBI terms and we record the RO BFO case as the only permissible exception.

Note, any given ontology can still be home to multiple namespaces/prefixes, as long as these belong the ontology in their entirety.

**Reply by balhoff** [reply]
I generally agree, but I think OMO might be another example.

**Reply by matentzn** [reply]
OMO is another exception :)

**Comment by matentzn** [comment]
Note to self: also look at Proper Names Ontology (PNO) and D-Acts

**Reply by pfabry** [reply]
I understand that PNO reuse terms from IAO, such as "personal name" (IAO:0020015) and its children. cf. https://github.com/information-artifact-ontology/IAO/issues/237
